libs/shared/account/account/src/lib/account.manager.ts

```diff
@@ -16,14 +16,17 @@ export class AccountManager {
   async createAccountStub(
     email: string,
     verifierVersion: number,
-    locale: string
+    locale: string,
+    clientSalt?: string
   ) {
-    const [emailCode, authSalt, kA, wrapWrapKb] = await Promise.all([
-      randomBytesAsync(16),
-      randomBytesAsync(32),
-      randomBytesAsync(32),
-      randomBytesAsync(32),
-    ]);
+    const [emailCode, authSalt, kA, wrapWrapKb, wrapWrapKbVersion2] =
+      await Promise.all([
+        randomBytesAsync(16),
+        randomBytesAsync(32),
+        randomBytesAsync(32),
+        randomBytesAsync(32),
+        randomBytesAsync(32),
+      ]);
```

Comment:
wait, this function isn't used anywhere?  now it feels like a time bomb with the generated wrapWrapKb2.

Reply:
Okay, agreed. I guess this will be used in the future. Let me make sure we don't have a warpWrapKbVersion2 value written here. I think since it's a stub it should be overwritten anyways, but you are correct, not creating a value at all would be much safer.